tests/codegen/issues/issue-86106.rs

```diff
@@ -10,11 +10,11 @@
 #[no_mangle]
 pub fn string_new() -> String {
     // CHECK-NOT: load i8
-    // CHECK: store i{{32|64}}
+    // CHECK: store i64
```

Comment:
If you want to not handle 32-bit here, I think you need to add `// only-64bit` to the header comments.

But it doesn't look like it's *that* bad to just support both here -- no shifts by `usize::BITS - 1` or anything -- so I'd encourage just leaving the regex matches in to work on both bitwidths.

Reply:
Hah. This went through several rebases. Looks like I can revert the test now.